src/main/java/ru/jru/quest/karaganov/service/FilterService.java

```diff
@@ -0,0 +1,28 @@
+package ru.jru.quest.karaganov.service;
+
+import lombok.NoArgsConstructor;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.util.HashMap;
+import java.util.Map;
+
+@NoArgsConstructor
```

Comment:
it seems that we can deal with the default constructor. In this case, why do we need this annotation?

Reply:
fixed